src/views/DisplayError.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div>
+    <v-alert type="error" color="pink darken-4">
+      <v-row align="center">
+        <v-col class="grow">{{ this.errorMessage }}</v-col>
+        <v-col class="shrink"> </v-col>
+      </v-row>
+    </v-alert>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "DisplayError",
+  computed: {
+    errorMessage: function() {
+      let message = "";
+      switch (this.$route.params.id) {
+        case 1:
```

Comment:
As before --- `1` is a "magic number" here. It doesn't mirror any HTTP code (unlike 403 and 404), just a "random" value. It would be fine if it was named, but seeing it in the caller code looks like this `1` has some meaning (possibly "the first one" or "some positive integer").